database/models/feedbacks.db.php

```diff
@@ -0,0 +1,26 @@
+<?php
+use Doctrine\ORM\Mapping\Column;
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\GeneratedValue;
+use Doctrine\ORM\Mapping\Id;
+use Doctrine\ORM\Mapping\ManyToOne;
+use Doctrine\ORM\Mapping\Table;
+
+
+#[Entity, Table(name: 'feedbacks')]
+class Feedback
+{
+    #[Id, Column, GeneratedValue]
+    public int|null $id = null;
+
+    #[ManyToOne(targetEntity: User::class)]
+    public User|null $user = null;
+
+    #[Column]
+    public string $description = "";
+
+    function __construct()
+    {
+    }
```

Comment:
You can remove the constructor if it's empty
```suggestion
```

Reply:
Done